### PR TITLE
PhpdocScalarFixer - fix property annotation too

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

**Comment by keradus** [comment]
Ref #1295

File: Symfony/CS/Fixer/Symfony/PhpdocScalarFixer.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ class PhpdocScalarFixer extends AbstractFixer
      *
      * @var array
      */
-    private static $tags = array('param', 'return', 'type', 'var');
+    private static $tags = array('param', 'return', 'type', 'var' 'property');
```
Comment (keradus) [inline Symfony/CS/Fixer/Symfony/PhpdocScalarFixer.php]: missing a comma

**Comment by GrahamCampbell** [comment]
This is kind of more of a feature than a bug fix I think @keradus. Wondering if maybe 1.10 is a better target?

**Comment by GrahamCampbell** [comment]
Also, we were looking for a place for me to update my email address across the code base, could I do that here @keradus?

**Comment by keradus** [comment]
I agree it may be treaten like enhance or bugfix.
But it was intended to fix long form to short form of types in phpdocs, we just missed that one annotation. Personally it's a bugfix to me.

**Comment by GrahamCampbell** [comment]
> Personally it's a bugfix to me.

Sure. That's what I first thought too when I selected the 1.9 branch. :)

**Comment by gharlan** [comment]
:+1:

**Comment by SpacePossum** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @GrahamCampbell.

**Pull request closed without merging**